views/pages/documents/meetings.pug

```diff
@@ -0,0 +1,45 @@
+extends /views/templates/frontend-base.pug
+
+append styles
+  link(href='/assets/css/pages/documents/meetings.css' rel="stylesheet")
+
+append scripts
+  script(src='/assets/js/meetings.js' defer)
+
+block body
+  +section(null, "white")
+    h1 Meetings
+    p!=data.descriptionMarkdown
+    - const meetingsByTerm = data.meetingGroups.reduce((prev, current) => {
+    -   for(const meeting of current.meetings) {
+    -     meeting.type = current.type;
+    -     if(prev[meeting.term]) {
+    -       prev[meeting.term].push(meeting);
+    -     } else {
+    -       prev[meeting.term] = [meeting];
+    -     }
+    -   };
+    -   return prev;
+    - }, {});
+
+    - 
+      const sortedMeetings = Object.entries(meetingsByTerm)
```

Comment:
The `-` above it does fix the problem! otherwise each JS line gets interpreted individually, and that breaks the function chaining.

the term numbers should always increase. the first digit increases by based on what 100s of years since 1900 (always increasing), the second and third digits are `currentYear % 100` (always increasing faster than the first digit), and the last digit is the first month of the term (always increasing faster than the 2nd/3rd digit).

(source: https://uwaterloo.ca/engineering/undergraduate-students/academic-support/term-information)